rebar.config

```diff
@@ -40,7 +39,7 @@
 
 {deps,
   [
-   {logi, ".*", {git, "git://github.com/sile/logi.git", {tag, "0.5.6"}}}
+   {logi, ".*", {git, "http://github.com/sile/logi.git", {tag, "0.5.7"}}}
```

Comment:
logi 0.5.7 has problems around erl_enno:anno(). (see https://github.com/sile/logi/pull/4 )
However, gen_cop does not use parse_transform and thus logi 0.5.7 works ok.
If needed, please update the tag here after logi is updated.:bow: